src/gui/MainWindow.cc

```diff
@@ -906,7 +906,7 @@ void MainWindow::loadViewSettings(){
 void MainWindow::loadDesignSettings()
 {
   QSettingsCached settings;
-  if (settings.value("design/autoReload", true).toBool()) {
+  if (settings.value("design/autoReload", false).toBool()) {
```

Comment:
@kintel what do you think, shall we flip the default? We could go back if we ever get a working cancel button :smile:.

Reply:
I feel default auto-reload is a bit tricky until we have a working way to cancel, so I'd vote to make it off by default.